assets/js/bf_woo_general_settings.js

```diff
@@ -221,6 +221,7 @@ jQuery(document).ready(function ($) {
             var regularPrice = general_settings_param.product_regular_price;
             if (regularPrice === "hidden") {
                 hide_general_regular_price = true;
+                $('#_regular_price').val(general_settings_param.regular_price_amount);
```

Comment:
@victorveas15 please add a validation for the next vars, to js errors.
- regular_price_amount
- sales_price_amount
- product_sales_start_date
- product_sales_end_date

Reply:
done